RecoEgamma/EgammaTools/interface/HGCalIsoProducer.h

```diff
@@ -0,0 +1,89 @@
+/*
+ * PhotonIsoProducer.h
+ *
+ *  Created on: 13 Oct 2017
+ *      Author: jkiesele
+ */
+
+#ifndef EGAMMATOOL_EGAMMAANALYSIS_HGCALISOPRODUCER_H_
+#define EGAMMATOOL_EGAMMAANALYSIS_HGCALISOPRODUCER_H_
```

Comment:
Please use an include guard matching the full file name subsystem_package_fileName_h